.github/workflows/cppcheck.yml

```diff
@@ -0,0 +1,33 @@
+name: cppcheck
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: read
+  pull-requests: read
+
+jobs:
+  cppcheck:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: checkout_repository
```

Comment:
Rewrite to `Checkout repository` to be consistent among other files

Reply:
Good catch! Fixed. 
(There is the same problem in `checkpatch.yml` too.)

Reply:
https://github.com/utat-ss/finch-flight-software/issues/63